Makefile

```diff
@@ -428,25 +428,75 @@ validate-codeserver-image: bin/kubectl
 		fi; \
 	done ; \
 
+# R Script (test.R)
+define R_SCRIPT
+	# Set a seed for reproducibility
+	set.seed(123)
+
+	# Generate a random dataset
+	x <- rnorm(100)
+	data <- data.frame(
+	x = x,
+	y = 2 * x + rnorm(100)
+	)
+	if ("x" %in% colnames(data) && is.numeric(data$x) &&
+		"y" %in% colnames(data) && is.numeric(data$y)) {
+	# Compute basic statistics
+	mean_x <- mean(data$x, na.rm = TRUE)
+	mean_y <- mean(data$y, na.rm = TRUE)
+	correlation <- cor(data$x, data$y, use = "complete.obs")
+	# Print the statistics
+	cat("Mean of x:", mean_x, "\n")
+	cat("Mean of y:", mean_y, "\n")
+	cat("Correlation between x and y:", correlation, "\n")
+
+	} else {
+	cat("Error: Columns 'x' and 'y' must be numeric and exist in the dataset.\n")
+	}
+endef
```

Comment:
It would be best if we bring this out of Makefile
and move it to `test_something.R` file

as it would easy to maintain that , then R code in Makefile.

Reply:
Agree, we may use `curl -sSL -o test.R "https://raw.githubusercontent.com/.../test.R"` 
Where we can host the test.R on `/rstudio/c9s-python-3.9/test/test.R` as we do for the Jupyter notebooks